### ISSUE TITLE
KeePass import: notes with multiple lines are not supported

### ISSUE THREAD

**Issue by fcturner** [fcturner/passwords#77]
Content of a testfile that fails on importing due to values not having a quotation mark (line 2):
```
"Account","Login Name","Password","Web Site","Comments"
"Sample Entry","User Name","Password","http://keepass.info/","Notes line 1

Notes line 3
Notes line 4"
"Sample Entry 2","User Name 2","Password 2","http://keepass.info/","Notes line 1"

```

~~Possible solutions:~~
~~- Read text between two `"`s: when line ends wihout `"`, look for every next line until `"` is found (only on notes)~~
~~- First convert all carriage returns / line feeds to `\n` on whole file and convert back on actual import~~

**Comment by fcturner** [comment]
Solved:
```js
var lines = contents.split('\n');
```
should be
```js
var lines = contents.split('\r\n');
```

Will be in next release.

**Comment by fcturner** [comment]
Fixed as of [8.0.14](https://github.com/fcturner/passwords/releases/tag/8.0.14).
